src/main/java/edu/cornell/kfs/gl/CuGeneralLedgerConstants.java

```diff
@@ -6,6 +6,8 @@
 
 	public static final String ANNUAL_CLOSING_CHARTS_PARAM = "ANNUAL_CLOSING_CHARTS";
 	public static final String COLLECTOR_HEADER_RECORD_TYPE = "HD";
+	public static final String COLLECTOR_TRAILER_RECORD_TYPE = "TL";
+	public static final String COLLECTOR_FILE_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
These constants are only in use by the Collector processing, but represent literals currently in use by the existing GL Collector code. Given the desire to split out the Concur-related code from our other KFS code as much as possible, though, I will move these constants into the Concur space instead (and also move the "HD" constant from a prior Concur PR).